spec/models/reports/export_raw_spec.rb

```diff
@@ -259,4 +259,28 @@
       )
     end
   end
+
+  describe "with cross core order details" do
+    # Defined in spec/support/contexts/cross_core_context.rb
+    include_context "cross core orders"
+
+    describe "for order details from a cross core project" do
+      let(:order_detail) { facility.orders.first.order_details.first }
```

Comment:
Is this used somewhere?

Reply:
yes, it's referenced earlier in the test setup, see lines 11 + 17:
https://github.com/tablexi/nucore-open/pull/4349/files#diff-33016b022a54e97ac24c29b9a74ce43bb8c74d67a9ebffa4b87fa020b4584348R17